### DISCUSSION TITLE
Is the Colorado OAP resource limit tied to SSI limits?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#4209] [Q&A] [answered]
They're the same value ($2,000/$3,000). Can we find the legal code to see if it points the same place? If so we can swap out the parameter. @CalebPena

**Comment by CalebPena** [comment]
The OAP and SSI resource limits are not explicitly linked.

Per Kathleen:

> OAP is calculated on an individual basis, and the OAP resource limit aligns with the SSI resource limits for an individual: $2,000. The [Colorado regs](https://sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11264&fileName=9%20CCR%202503-5) don’t explicitly state that OAP resource limits must be the same as SSI resource limits. However, this is implicit in the fact that the same state regs govern both OAP and AND-CS (which aligns with SSI) and the requirement that OAP recipients pursue all available SSI benefits, with SSI being credited against a person’s OAP grant.
That said, I don’t think we can assume that if the SSI resource limit changes, the OAP limit will automatically change. The OAP limit is in the state regs and would require updating.

**Reply by MaxGhenis** [reply, marked as answer]
Found it - the Colorado regs are indeed downstream of [C.R.S. 26-2-111](https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=a19534a1-2a50-48b9-a1c8-c6800800960a&config=014FJAAyNGJkY2Y4Zi1mNjgyLTRkN2YtYmE4OS03NTYzNzYzOTg0OGEKAFBvZENhdGFsb2d592qv2Kywlf8caKqYROP5&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A662M-2CD3-GXF6-80SP-00008-00&pdcontentcomponentid=234176&pdteaserkey=sr9&pditab=allpods&ecomp=bs65kkk&earg=sr9&prid=8611c23e-3c5e-4423-bb96-ce9149909453), which ties the OAP resource limit to SSI:

> (2) Old age pension.
> (a)  Except as provided in paragraphs (c) and (d) of this subsection (2), public assistance in the form of the old age pension shall be granted to any person who meets the requirements of subsection (1) of this section and any one of the following requirements:
> (I) The person is a United States citizen or a qualified alien, has attained the age of sixty years or more, and **meets the resource eligibility requirements of the federal supplemental security income program**; or

Filed https://github.com/PolicyEngine/policyengine-us/issues/4220
